doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -71,7 +71,7 @@ Since OpenStack is a complex, multi-project framework, we initially will focus o
 
 The architecture consists of the services shown in the **Figure 4-1**; Ironic and Swift are optional OpenStack srevices. The rest of this document will address the specific Common Telco NFVI implementation requirements and recommendations.
 
-<p align="center"><img src="../figures/Figure_3_1_Core_NFVI_Services_v2.png" alt="Core NFVI Software Services" title="core NFVI Software Services" width="100%"/></p>
+<p align="center"><img src="../figures/Figure_3_1_Core_NFVI_Services_v3.png" alt="Core NFVI Software Services" title="core NFVI Software Services" width="100%"/></p>
```

Comment:
Some comment to the figure: 

- Do we want to show shich component is running on which node on this picture? If yes I would show it in a way that the services are inside the nodes
- I would add a box for common services, like databases, message bus and so on
- I would add a box for monitoring and one for alerting to the control and to the compute nodes.
- Do we require DPDK in this RA?
- If Cloud deployment is a different node, than it should be indicated. Similar to Logging and alerting and shared storage
- If services are not detailed in other services why they are in case of Keystone?

@pgoyal01 I can make a proposal if you link the source of the figure.

Reply:
@CsatariGergely Sending you the source figure. Earlier version had differentiated between control/compute but with some services replicated in both there was an ask for simplification.

Reply:
@pgoyal01 in which channel did you sent it?

Reply:
@CsatariGergely I had sent you an email with the source diagrams. Just resent the email.

Reply:
Tanks. What do you think about this: 
![image](https://user-images.githubusercontent.com/11458775/67658213-3c43e980-f959-11e9-9a04-9edb706d8cd5.png)
?

Reply:
@CsatariGergely Do we need to show the "Host OS" in "Control nodes"?

Reply:
@CsatariGergely - looks good - I would suggest we make it clear the boxes on the right span control + compute layers as they apply to both.

Where / how do we show central network nodes (CVR)?
Common services (databases. message queues) - suggest to add "OpenStack" as a prefix
Finally we were asked to remove KVM/QEMU under hypervisor as VMWare ESXi is a valid option

Reply:
@pgoyal01 The title of the slide is OpenStack Core services, therefore Host OS would not be needed. I still somehow think that it is an important piece of the RA, therefore I would rather keep Host OS and rename the figure to "Core services of the Reference Architecture"

@iangardner22 

- I hope we have a statement somewhere in the document stating that these are only roles which can be combined on the same hardware. 
- Will we have central network nodes?
- Difficult question. These are used by the different OpenStack services, but themselves are not OpenStack services (eg.: mysql or Rabbit MQ)
- We have component names everywhere in the figure, so we should have something . How about Qemu/KVM or ESXi?

Reply:
![image](https://user-images.githubusercontent.com/11458775/67687798-8355cd00-f9a1-11e9-810d-a53bff898883.png)
?

Reply:
@CsatariGergely 
- OK we'll need to do that (functional roles)
- for OVS kernel network model we will have network nodes yes
- yes my point was they are used by OpenStack but I am not overly worried
- I know Pankaj faced a challenge on this one...   @rabi-abdel ?

Reply:
> ![image](https://user-images.githubusercontent.com/11458775/67687798-8355cd00-f9a1-11e9-810d-a53bff898883.png)
> ?
Minor comments:
1. **Compute Nodes block**: Can we remove Common Services from here like databases, messaging etc ? DB and RabbitMQ are a part of common services in control node and are generally not present on compute nodes.
2. **Control Nodes block**: Can we add a bottom layer for **Host OS**, just like we are showing in compute nodes block ?
3. **Control Nodes block**:

Reply:
1. Okay, however I believed that RabbitMQ is there in the compute nodes.
2. Okay

![image](https://user-images.githubusercontent.com/11458775/67841322-1a349d80-faf8-11e9-8137-707510c1ae12.png)

Reply:
Thanks !!! Rabbit MQ is present only on Controller nodes.
Refer. https://docs.openstack.org/install-guide/environment-messaging-ubuntu.html

Reply:
@CsatariGergely , Rabbitmq should be part of compute and MariaDB/MySQL is part controllers

Reply:
@ASawwaf I don't think thats a standard OpenStack installation process ?
If you see [this](https://docs.openstack.org/install-guide/environment-messaging-ubuntu.html) and [this](https://docs.openstack.org/nova/queens/install/compute-install-ubuntu.html#install-and-configure-components), they refer to RabbitMQ installation only on controller nodes only. The compute nodes take the RabbitIP+Port to connect to rabbitMQ servers running on controller.
MOS installs rabbitMQ on controller nodes only.

Reply:
Okay, let's not have RabbitMQ on the compute nodes. 
@pgoyal01 I will send you the ppt with the diagram (as I can not upload it to here for some reason...)

Reply:
Looks good @CsatariGergely thanks.  Suggest we simplify "Common Services for OpenStack Services" to "Common Services for OpenStack" or "OpenStack Control Plane Services".

@rabi-abdel what did you want to do with ESXi as previously you suggested we kept Hypervisor generic in the RA and then made the RI specific?

Reply:
@iangardner22 what am I missing? I only see the diagram state "Common Services" under both Control and Compute nodes.

Reply:
> Common Services for OpenStack

I think we should not let the hypervisor question to be an RI decision. RA should define everyting what can affect the behaviour of the VNF and the hypervisor can.

Reply:
Simplifies services text: 
![image](https://user-images.githubusercontent.com/11458775/67963851-f52f5000-fc07-11e9-966a-1504bb512050.png)

Reply:
> @iangardner22 what am I missing? I only see the diagram state "Common Services" under both Control and Compute nodes.

This was a previous diagram - the word OpenStack was repeated.  It's fixed now